client/my-sites/drafts/author-selector.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import NavSegmented from 'components/section-nav/segmented';
+import NavItem from 'components/section-nav/item';
+import userLib from 'lib/user';
```

Comment:
`lib/user` is a deprecated module that we're in the process of moving away from. You should be able to pass a user object from [the `getCurrentUser` selector](https://github.com/Automattic/wp-calypso/blob/9853eb5f87cfe331140943385ba7b989a2833429/client/state/current-user/selectors.js#L18-L31).

Of less importance, we should consider enhancing the `<Gravatar />` component to accept a `userId` and look up the user in its own selector.